src/components/NcSelect/NcSelect.vue

```diff
@@ -1152,7 +1155,7 @@ body {
 }
 
 .vs__dropdown-menu {
-	border-color: var(--color-primary-element) !important;
+	border-color: var(--color-main-text) !important;
 	padding: 4px !important;
```

Comment:
```suggestion
	padding: 4px !important;
	box-shadow: none;
```
This would remove the `box-shadow` as @marcoambrosini suggested (ignore the misalignment of the dropdown, unrelated):

| Before | After |
|-|-|
| ![Screenshot 2023-11-03 at 11-46-12 Nextcloud Vue Style Guide](https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/0e72859a-74f4-4680-ad8d-e705b7e1742c) | ![Screenshot 2023-11-03 at 11-46-32 Nextcloud Vue Style Guide](https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/309f30da-ccb1-451e-972e-0e5f2ec04852) |

Reply:
Thank you!